.github/workflows/score_history.yml

```diff
@@ -69,6 +72,13 @@ jobs:
       - name: Install dependencies
         run: pip install numpy pandas requests python-dotenv GitPython sqlalchemy psycopg2
 
+      - name : Setup PostgreSQL tunnel to scalingo
+        env:
+          SCALINGO_REGION: ${{ secrets.SCALINGO_REGION }}
+          SCALINGO_APP: cdb-${{ matrix.service }}${{ secrets.APP_SUFFIX }}
+          SCALINGO_API_TOKEN: ${{ secrets.SCALINGO_API_TOKEN }}
+        run : echo "coucou"
+
```

Comment:
I don't think this should be part of this specific PR

Reply:
Oops ^^